src/tests/ftest/container/container_check.py

```diff
@@ -0,0 +1,144 @@
+        self.dfuse.get_params(self)
+
+        # update dfuse params
+        self.dfuse.set_dfuse_params(self.pool)
+        self.dfuse.set_dfuse_cont_param(self.container)
+        self.dfuse.set_dfuse_exports(self.server_managers[0], self.client_log)
+
+        try:
+            # start dfuse
+            self.dfuse.run(False)
+        except CommandFailure as error:
+            self.log.error("Dfuse command %s failed on hosts %s",
+                           str(self.dfuse),
+                           self.dfuse.hosts,
+                           exc_info=error)
+            self.fail("Test was expected to pass but it failed.\n")
+
+    def test_dfusecontainercheck(self):
+        """Jira ID: DAOS-3635.
+
+        Test Description:
+            Purpose of this test is to try and mount different container types
+            to dfuse and check the behavior.
+        Use cases:
+            Create pool
+            Create container of type default
+            Try to mount to dfuse and check the behaviour.
+            Create container of type POSIX.
+            Try to mount to dfuse and check the behaviour.
+        :avocado: tags=all,small,full_regression,dfusecontainercheck
+        """
+        # get test params for cont and pool count
+        cont_types = self.params.get("cont_types", '/run/container/*')
+
+        # Create a pool and start dfuse.
+        self.create_pool()
+
+        for cont_type in cont_types:
+            # Get container params
+            self.container = TestContainer(
+                self.pool, daos_command=DaosCommand(self.bin))
+            self.container.get_params(self)
+            # create container
+            if cont_type == "POSIX":
+                self.container.type.update(cont_type)
+            self.container.create()
+            try:
+                # mount fuse
+                self.start_dfuse()
+                # check if fuse got mounted
+                self.dfuse.check_running()
+                # fail the test if fuse mounts with non-posix type container
+                if cont_type == "":
+                    self.fail("Non-Posix type container got mounted over dfuse")
+            except CommandFailure as error:
+                # expected to throw CommandFailure exception for non-posix type
+                # container
+                if cont_type == "":
+                    self.log.info("Expected behaviour: Default container type \
+                        is expected to fail on dfuse mount: %s", str(error))
+                # fail the test if exception is caught for POSIX type container
+                elif cont_type == "POSIX":
+                    self.log.error("Posix Container dfuse mount \
+                        failed: %s", str(error))
+                    self.fail("Posix container type was expected to mount \
+                        over dfuse")
+            # stop fuse and container for next iteration
+            if not cont_type == "":
```

Comment:
Suggest to create a variable DEFAULT_CONT_TYPE=<Default container type> and use it here and other places where container type = "" is specified.